src/collection.rs

```diff
@@ -2,20 +2,28 @@ use std::sync::Arc;
 
 use serde::{
     de::{Deserializer, Error as DeError},
-    Deserialize,
+    Deserialize, Serialize,
 };
 use serde_json::json;
 use url::Url;
 
 use maybe_async::maybe_async;
 
-use crate::client::ClientExt;
-use crate::response::ArangoResult;
-use crate::{response::serialize_response, ClientError};
+use crate::{
+    client::ClientExt,
+    response::{deserialize_response, ArangoResult},
+    ClientError,
+};
 
 use super::{Database, Document};
+use crate::document::{
+    DocumentHeader, DocumentInsertOptions, DocumentOverwriteMode, DocumentReadOptions,
+    DocumentRemoveOptions, DocumentReplaceOptions, DocumentResponse, DocumentUpdateOptions,
+};
+use http::Request;
+use serde::de::DeserializeOwned;
 
-#[derive(Debug, Deserialize)]
+#[derive(Deserialize)]
```

Comment:
Sorry.. I did not explain my point clear enough.

I don't mean to strip these `Debug` trait for all these field. 

What I against is that add `Debug` trait as a mandatory requirement when using generic types:

```rust
fn some_function<T>(input:T) where T: Debug{}
```

Thank you again!!!